pubsub/google/cloud/pubsub_v1/publisher/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+from google.cloud.pubsub_v1.publisher.client import Client
```

Comment:
I discussed with @dhermes and he suggested that we use `Client` in the hand-written portions wherever possible, since some of the GAPIC client names are obnoxious.

Not married to this if we want to move in the other direction.